fabfile.py

```diff
@@ -231,21 +231,25 @@ def help(name):
 @task
 def production():
     """Select production environment"""
+    env['environment'] = 'production'
```

Comment:
https://github.com/alphagov/fabric-scripts/blob/master/campaigns.py#L35

We seem to use both in different places, not sure if there is a preference. *snap* with the Python-fu.